app/models/miq_server.rb

```diff
@@ -544,4 +545,16 @@ def zone_unchanged_in_pods
 
     errors.add(:zone, N_('cannot be changed when running in containers')) if zone_id_changed?
   end
+
+  def messaging_health_check
+    require 'manageiq-messaging'
```

Comment:
Do we need to require this here if it is required in `MiqQueue.messaging_client`?

Reply:
Nope thanks for the catch, I think this is a remnant from when I was just starting out and wasn't using `MiqQueue` initially